packages/Algernon.package/ALGMethodType.class/class/algertemListWith..st

```diff
@@ -1,4 +1,10 @@
 algertem lists
 algertemListWith: aClass
-	^ aClass selectors asOrderedCollection
-		collect: [:each | ALGItem withAlgertype: (self withType:each withClass:aClass) ]
\ No newline at end of file
+	| items |
+	items := aClass selectors asOrderedCollection
+		collect: [:each | ALGItem withAlgertype: (self withType: each withClass: aClass) ].
+		
+	items addAll: (aClass class selectors asOrderedCollection
+		collect: [:each | ALGItem withAlgertype: (self withType: self prefixForClassMethods, each withClass: aClass class) ]).
```

Comment:
> `aClass class`

Wow, that is odd :laughing: (but correct)

Reply:
Yeah, quite strange 😄 But as far as my understanding goes `aClass class` then refers to the meta-class of `aClass` (which internally handles the method dictionary for the class side of `aClass`).